### ISSUE TITLE
Add more options for dropdown menu content side props

### ISSUE THREAD

**Issue by majidsajadi** [radix-ui/primitives#1359]
### Overview

Dropdown menu content side prop supports 'left', 'right', 'top', 'bottom'. i guess some more options such as 'bottom-left', 'bottom-right', etc would be much appreciated.

### Examples in other libraries

[Ant Dropdown](https://ant.design/components/dropdown/#Dropdown)

### Who does this impact? Who is this for?

I guess my use case can be a good example why this feature is useful. i use this component for my new app. i added a menu in my side bar. for now i use 'bottom' option but a part of the menu overlay is outside of the side bar which is not the perfect option for me.

**Comment by benoitgrelard** [comment]
Hi @majidsajadi, I think the examples you are suggesting are conflated axis basically.
In our case you use a combination of `side` and `align` props, so for example your `bottom-left` would be achieved using `side="bottom"` + `align="start"`.

Hope that makes sense!

✌️

**Comment by enesozturk** [comment]
Hey @benoitgrelard, what would be the configs to make the content the same width of the trigger? It'll be like `side="left-to-right"`. Is this possible with the current version or it'll require some tricks? I couldn't find the proper props to do that.

**Comment by benoitgrelard** [comment]
@enesozturk https://www.radix-ui.com/docs/primitives/components/dropdown-menu#constrain-the-contentsub-content-size

**Comment by enesozturk** [comment]
@benoitgrelard, thanks for the response. I'm trying to make it work for one hour but that prop doesn't seems to work. I'm wondering if I should set another value or props for this. Just the setting class name and giving the provided styles will do that? I see in the inspector that the class name and styles are applied, but the width is the same.

Another thing, I realized is that my use case might be closer to the Select component. So I tried it as well. It's not even rendering content, the positions of the content are'nt calculated so it's rendered without position and cannot be seen in the screen. I know this is something I need to provide reproduce examples & code snippets, the issue is highly possible on my side. What should I be sure about before using the Select component? Why it might not be calculating the position as expected?
